src/data/alumni.ts

```diff
@@ -0,0 +1,127 @@
+export interface Alumni {
+  name: string;
+  description: string;
+}
+
+export const alumni: Alumni[] = [
+  {
+    name: 'Khanh Nguyen',
+    description: `Khanh is a Junior from Vietnam studying Computer Science. BROCKHAMPTON is his all-time favorite boyband. Outside of school, he enjoys trying out good food, exploring the city and contributing to projects. 
+    Khanh was the former president.`,
+  },
+  {
+    name: 'Aaditya Mehta',
+    description: `Aaditya is a Senior studying Economics and Mathematics, Aaditya spends way too much time in Stern (#IAmStern). In his free time he likes to listen to music, take long walks, and read Econ papers, sometimes simultaneously. 
+    Aaditya was the former treasurer.`,
+  },
+  {
+    name: 'Sonia Jain',
+    description: `I'm Sonia and I'm a junior at New York University double majoring in Computer Science and Media, Culture, and Communications. I am interested in game design, art, and I love animals. 
+    In my free time, I work with a variety of organizations to help find dogs and cats' homes. I also love painting and digital art so when I'm not working with animals, I am creating. Sonia was the former director of marketing & communications.`,
+  },
+  {
+    name: 'Bruce Jiang',
+    description: `Bruce is a junior from China studying Computer Science and Mathematics. Outside of class, he is crazy in soccer (especially in Chelsea) and basketball. He spends tons of time playing FIFA (most of the time playing Chelsea). 
+    Besides, he likes listening to music, reading about history, and travelling. He was the former secretary.`,
```

Comment:
```suggestion
    description: `Bruce is a junior from China studying Computer Science and Mathematics. Outside of class, he is crazy in soccer (especially in Chelsea) and basketball. He spends tons of time playing FIFA (most of the time playing Chelsea). Besides, he likes listening to music, reading about history, and traveling. Bruce was a former secretary.`,
```
Fix typo and refer by name